velox/functions/prestosql/JsonFunctions.h

```diff
@@ -74,4 +74,25 @@ struct JsonArrayLengthFunction {
   }
 };
 
+template <typename T>
+struct JsonArrayContainsFunction {
+  template <typename TOutput = bool, typename TInput1, typename TInput2>
```

Comment:
Does TOutput have to be a template variable ? Especially since its always bound to boolean.

Even TInput1 doesn't need to be templatized as its always a varchar json. A template variable is only needed for the 2nd input parameter.

Reply:
The following error appears if only the 2nd input parameter is templated: 
```In template: static_assert failed due to requirement 'udf_has_call || udf_has_callNullable || udf_has_callNullFree' "UDF must implement at least one of `call`, `callNullable`, or `callNullFree`".```

Reply:
Change it like this:
```
template <typename T>
struct JsonArrayContainsFunction {
  VELOX_DEFINE_FUNCTION_TYPES(T);

  template <typename TInput>
  FOLLY_ALWAYS_INLINE bool
  call(bool& result, const arg_type<Varchar>& json, const TInput& value) {
    ...
  }
```
I tested locally. This should compile.

Reply:
Thanks for sharing, modified accordingly, using ``const arg_type<TInput>& value`` had resulted in the error where the call method for the registered functions couldn't be found.